app/controllers/api/v1/cards_controller.rb

```diff
@@ -74,7 +74,7 @@ def format_search_params
     #   ["draven"]
     # ]
     attributes_from_query = params[:query].scan(
-      /((\w+:".*?"|\w+:\w+)?(\w+:".*?"|\w+:\w+)|\w+)/
+      /((\w+:".*?"|\w+[:<>]=?\w+)?(\w+:".*?"|\w+:\w+|\w+[<>=]=?\d+)|\w+\b)/
```

Comment:
I feel like if anything deserved comments to explain what it was doing, it is your regex. lol I won't pretend to understand the regex so well, but testing it out, it works great.